iree/compiler/Translation/SPIRV/init_translations.h

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef IREE_COMPILER_TRANSLATION_SPIRV_INIT_TRANSLATIONS_H_
+#define IREE_COMPILER_TRANSLATION_SPIRV_INIT_TRANSLATIONS_H_
+
+#include "iree/compiler/Translation/SPIRV/IndexComputation/IndexComputationPass.h"
+#include "iree/compiler/Translation/SPIRV/LinalgToSPIRV/Passes.h"
+#include "iree/compiler/Translation/SPIRV/Passes/Passes.h"
+#include "iree/compiler/Translation/SPIRV/XLAToSPIRV/IREEToSPIRVPass.h"
+#include "mlir/Pass/Pass.h"
```

Comment:
Internal checks want this include to have a matching dependency (probably `"@llvm-project//mlir:Pass",` in Bazel). I'm looking to disable that check and apply the fixes it suggests on-demand instead, but it's required for now.

Reply:
No, the header is included, that's fine. Added the dep.